static/app/views/dashboardsV2/controls.tsx

```diff
@@ -90,18 +100,41 @@ class Controls extends React.Component<Props> {
       <StyledButtonBar gap={1} key="controls">
         <DashboardEditFeature>
           {hasFeature => (
-            <Button
-              data-test-id="dashboard-edit"
-              onClick={e => {
-                e.preventDefault();
-                onEdit();
-              }}
-              priority="primary"
-              icon={<IconEdit size="xs" />}
-              disabled={!hasFeature}
-            >
-              {t('Edit Dashboard')}
-            </Button>
+            <React.Fragment>
+              <Button
+                data-test-id="dashboard-edit"
+                onClick={e => {
+                  e.preventDefault();
+                  onEdit();
+                }}
+                priority="primary"
+                icon={<IconEdit size="xs" />}
+                disabled={!hasFeature}
+              >
+                {t('Edit Dashboard')}
+              </Button>
+              <Feature
+                organization={organization}
+                features={['organizations:issues-in-dashboards']}
+              >
+                {({hasFeature: hasIssuesInDashboardsFeature}) =>
+                  hasIssuesInDashboardsFeature && (
+                    <Button
+                      data-test-id="dashboard-add-issues-widget"
+                      onClick={e => {
+                        e.preventDefault();
+                        onAddIssueWidget();
+                      }}
+                      priority="primary"
+                      icon={<IconAdd size="xs" />}
+                      disabled={!hasFeature}
+                    >
+                      {t('Add Issue Widget')}
+                    </Button>
+                  )
+                }
```

Comment:
same feedback i had for my widget library PR, we should make sure that we don't exceed the max widgets either by disabling this or disabling the add widget button (can be follow up)